shared/chat/conversation/input-area/normal/shared.js

```diff
@@ -1,9 +1,54 @@
 // @flow
 import * as React from 'react'
-import {Meta} from '../../../../common-adapters'
-import {globalColors, platformStyles, styleSheetCreate} from '../../../../styles'
+import * as I from 'immutable'
+import * as Kb from '../../../../common-adapters'
+import * as Styles from '../../../../styles'
 import {formatDurationShort} from '../../../../util/timestamp'
 
+export const isTyping = (typing: I.Set<string>) => (
```

Comment:
instead of making a helper that returns a component we should just make this a component. aka
```
const IsTyping = ({typing: I.Set<string>}) => <Kb.Text>...
```

Reply:
then you can just do 
```
<IsTyping typing={this.props.typing} /> above
```